### PR TITLE
Changelog fixes.

### PR DESCRIPTION
**Pull request by godai78**
### Changelog initial review.

Closes cksource/ckeditor5-internal#417

### CODE REVIEW

**Review by AnnaTomanek — CHANGES_REQUESTED** [review]
The changelog is missing the entire first (and most important) section. Check the highlights for the previous versions. And the iteration summary is here for inspiration: #8384

cc @pomek If we are missing this step in the instructions, it should probably be added.

**Comment by godai78** [comment]
Added those. Please review if my choice was the best one there is.

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,20 @@ Changelog
 
 ### Release highlights
 
+We are happy to announce the release of CKEditor 5 v24.0.0.
+
+This release brings the new, exciting features to name just [accessible input labels](https://github.com/ckeditor/ckeditor5/issues/1098) and the long-awaited [multiline toolbar](https://github.com/ckeditor/ckeditor5/issues/6146).
+
+[Adding features to builds](https://github.com/ckeditor/ckeditor5/issues/8395) is now available as well as [simply pasting an image URL](Ihttps://github.com/ckeditor/ckeditor5/issues/8236) to put and image into the content.
```
Comment (pomek) [inline CHANGELOG.md:12]: > Adding features to builds

This is not ready yet.

Comment (godai78) [reply CHANGELOG.md:12]: OK, so I'll remove it.

**Review by AnnaTomanek — APPROVED** [review]
I reviewed the changelog and rewrote the highlights part, based on the previous format (see 2 releases back). Let's leave the prose for the blog post and stick to a more dry tone (list) in the changelog.

**Comment by godai78** [comment]
Seem the poetry is only in the code, alas.

**Comment by godai78** [comment]
No link for inserting images?

**Comment by AnnaTomanek** [comment]
Good catch, fixed that now. I screwed the merge.

**Pull request merged**